tools/analyse-storage/src/main.rs

```diff
@@ -0,0 +1,282 @@
+use clap::Parser;
+use near_store::db::RocksDB;
+use near_store::{col_name, DBCol};
+use nearcore::get_default_home;
+use once_cell::sync::Lazy;
+use rayon::prelude::*;
+use rocksdb::{ColumnFamilyDescriptor, Options, DB};
+use std::collections::HashMap;
+use std::path::PathBuf;
+use std::sync::{Arc, Mutex};
+use std::{panic, println};
+use strum::IntoEnumIterator;
+
+static DEFAULT_HOME: Lazy<PathBuf> = Lazy::new(get_default_home);
+
+#[derive(Parser)]
+struct Cli {
+    #[clap(long, value_parser, default_value_os = DEFAULT_HOME.as_os_str())]
+    home: PathBuf,
+
+    #[arg(short, long)]
+    column: Option<String>,
+
+    #[arg(short, long)]
+    limit: Option<usize>,
+}
+
+pub fn get_db_col(col: &str) -> Option<DBCol> {
+    match col {
+        "col0" => Some(DBCol::DbVersion),
+        "col1" => Some(DBCol::BlockMisc),
+        "col2" => Some(DBCol::Block),
+        "col3" => Some(DBCol::BlockHeader),
+        "col4" => Some(DBCol::BlockHeight),
```

Comment:
You don't need these arms anymore, `col_name(db_col) == col` now handles all match cases, right?

Reply:
Sorry I forgot to do that